hadoop-hdds/docs/content/interface/S3.md

```diff
@@ -116,7 +116,7 @@ aws s3api --endpoint http://localhost:9878 create-bucket --bucket bucket1
 
 To show the storage location of a S3 bucket, use the `ozone s3 path <bucketname>` command.
 
-```
+```bash
 aws s3api --endpoint-url http://localhost:9878 create-bucket --bucket=bucket1
 
 ozone s3 path bucket1
```

Comment:
Can you confirm the output volume name is "s3thisisakey"?

Reply:
No, it looks more like some random string:

```
Volume name for S3Bucket is : s3100b8cad7cf2a56f6df78f171f97a1ec
Ozone FileSystem Uri is : o3fs://bucket1.s3100b8cad7cf2a56f6df78f171f97a1ec
```

Reply:
I think "s3thisisakey" is just some random value for document.

Reply:
To be more precious: the volume name is "s3" + AWS_ACCESS_KEY_ID (it's not random, but your personal id, I guess)

This examples assumes that 

```
export AWS_ACCESS_KEY_ID=thisisakey
```
is set.

Reply:
But, yes, this is just an example, could be any other string. And this convention can be changed later, so we shouldn't share it in the doc IMHO.

Reply:
Actually its some encoded form of the access key ID, which I set to `any`.  For `thisisakey` it turns out to be `6c7412f65c83a118bafff79dd270186c`.